packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -55,22 +55,28 @@ function ScreenRoot() {
 							</HStack>
 						</NavigationButton>
 					) }
+					<ContextMenu />
 				</VStack>
 			</CardBody>
 
-			<CardBody>
-				<ContextMenu />
-			</CardBody>
-
 			<CardDivider />
 
 			<CardBody>
+				<Spacer
+					as="p"
+					paddingTop={ 2 }
+					/*
+					 * 13px matches the text inset of the NavigationButton (12px padding, plus the width of the button's border).
+					 * This is an ad hoc override for this particular instance only and should be reconsidered before making into a pattern.
+					 */
+					paddingX="13px"
+					marginBottom={ 4 }
+				>
+					{ __(
+						'Customize the appearance of specific blocks for the whole site.'
+					) }
+				</Spacer>
 				<ItemGroup>
-					<Item>
-						{ __(
-							'Customize the appearance of specific blocks for the whole site.'
-						) }
-					</Item>
 					<NavigationButton path="/blocks">
```

Comment:
The use of `Item` here was problematic because it ignores the underlying HTML semantics (`list` and `listitem`).

Maybe a reminder for us to name components in a way that sufficiently evokes the HTML semantics? Another instance of semantic misuse is addressed in #40590.

Reply:
> while in our UIs, not everything that gets grouped together is necessary a list.

I would probably argue that anything that's visually grouped together using `ItemGroup` _is and should be_ a list. The ways I saw it misused (in Global Styles at least) were not legitimate use cases. So I still think it's a reasonable default for it to have list semantics.

> - we could have named the component more explicitly (i.e. `ItemList`)

This, absolutely. But also, I kind of think we should switch to using `as="ul"` instead of `role="list"`. Then it would be possible to cleanly [remove the semantics](https://github.com/WordPress/gutenberg/pull/40590#discussion_r857694051) (however inadvisable that may be). As a plus, people could also make it an `ol` if they wanted!

Reply:
> I would probably argue that anything that's visually grouped together using `ItemGroup` _is and should be_ a list. The ways I saw it misused (in Global Styles at least) were not legitimate use cases. So I still think it's a reasonable default for it to have list semantics.

In principle I agree with this. The only worry that I have is to avoid coupling semantics and visual look, but the rest of your reply also partially addresses that.

> > * we could have named the component more explicitly (i.e. `ItemList`)
> 
> This, absolutely. But also, I kind of think we should switch to using `as="ul"` instead of `role="list"`. Then it would be possible to cleanly [remove the semantics](https://github.com/WordPress/gutenberg/pull/40590#discussion_r857694051) (however inadvisable that may be). As a plus, people could also make it an `ol` if they wanted!

This all makes sense. I'd be curious to know the original reasoning for using `role` instead of HTML elements in `ItemGroup` (@griffbrad maybe you have any context here?)

Reply:
There was a conversation between @ItsJonQ and @diegohaz about the semantics of `ItemGroup` when it was first introduced:
https://github.com/WordPress/gutenberg/pull/30097#issuecomment-804394902